challenge-1/bmi-comparator.js

```diff
@@ -0,0 +1,18 @@
+class Person {
```

Comment:
Hello can we add some documentation for the code [here](https://devhints.io/jsdoc) and [here](https://jsdoc.app/index.html) are some examples

Reply:
done

Reply:
done